flows/events/contact_property_changed.go

```diff
@@ -38,6 +38,16 @@ func NewContactPropertyChangedEvent(property string, value string) *ContactPrope
 // Type returns the type of this event
 func (e *ContactPropertyChangedEvent) Type() string { return TypeContactPropertyChanged }
 
+// AllowedOrigin determines where this event type can originate
```

Comment:
Instead of repeating these can't we embed proper base types that implement these?

Reply:
updated